modules/promotions/assets/js/editor/e-promotion/app/app.js

```diff
@@ -0,0 +1,61 @@
+import { DirectionProvider, LocalizationProvider, ThemeProvider } from '@elementor/ui';
+
+import Infotip from '@elementor/ui/Infotip';
+import Chip from '@elementor/ui/Chip';
+import Button from '@elementor/ui/Button';
+import Card from '@elementor/ui/Card';
+import CardHeader from '@elementor/ui/CardHeader';
+import CardMedia from '@elementor/ui/CardMedia';
+import CardContent from '@elementor/ui/CardContent';
+import CardActions from '@elementor/ui/CardActions';
+import Typography from '@elementor/ui/Typography';
+
+const InfoTipCard = () => {
+	return (
+		<Card sx={ { maxWidth: 300 } }>
+			<CardHeader title={
+				<>
+					Bring Headlines to Life
+					<Chip size="small" label="pro" variant="outlined" color="promotion" />
+				</>
+			} />
```

Comment:
there must be a better way to arrange the header...
but for some reason adding "children" didn't work (wasn't looking close enough)

wanted something like:
```
<CardHeader>text <Chip/> <CloseButton /></CardHeader>
```